src/structs.jl

```diff
@@ -479,4 +484,6 @@ Base.@kwdef struct Model
     volume_units::Maybe{String} = nothing
     notes::Maybe{String} = nothing
     annotation::Maybe{String} = nothing
+    sbo::Maybe{String} = nothing
+    cv_terms::Vector{CVTerm} = []
```

Comment:
You may want to add these two fields to the tests in https://github.com/LCSB-BioCore/SBML.jl/blob/0eb0a446b62b5072e851d61fcab1e2e0c66b8124/test/writemodels.jl#L58

Reply:
I actually systematized it to whole model, that way it's much easier.

Reply:
Uh, yeah, good idea.  I think I added each field manually there before having the smarter custom `==` method for the other structs.